graphistry/feature_utils.py

```diff
@@ -1096,8 +1079,7 @@ def process_nodes_dataframes(
     text_cols: List[str] = []
     text_model: Any = None
     text_enc = pd.DataFrame([])
-    has_deps_text, import_text_exn, _ = lazy_import_has_dependancy_text()
-    if has_deps_text and (feature_engine in ["torch", "auto"]):
+    if deps.sentence_transformers and (feature_engine in ["torch", "auto"]):
```

Comment:
yeah i think we're being a bit fast-and-loose: sometimes we specifically want some st, sometimes we should be checking if the collection of deps_text are all available